metastor/managedObjectLoader/src/main/workflow-def/singletonObjectsAddPartitionWorkflow.bpmn

```diff
@@ -19,23 +19,23 @@
     <scriptTask id="prepareEmailContent" name="Prepare Email Content" scriptFormat="javascript" activiti:autoStoreVariables="false">
       <script>
       var emailMessage="";
-      if (createCluster_taskStatus == 'ERROR') { 
```

Comment:
Has anything changed in this file?

Reply:
yeah, disregarding whitespace, there are some small changes, like ";"